bang/magicollections.py

```diff
@@ -1899,22 +1899,15 @@ def to_fields(self, view, item, *args, **kwargs):
         setSubField(fields, 'unlock', key='value', value=item.unlock_sentence)
 
         for difficulty, verbose_name in models.Song.DIFFICULTIES:
-            note_image = u'{}img/note.png'.format(RAW_CONTEXT['static_url'])
-            image = lambda f: u'{static_url}img/songs/{difficulty}.png'.format(
-                static_url=RAW_CONTEXT['static_url'],
-                difficulty=difficulty,
-            )
-            diff = getattr(item, u'{}_difficulty'.format(difficulty), None)            
-            setSubField(fields, u'{}_notes'.format(difficulty), key='image', value=image)            
-            if diff != None:
+            note_image = staticImageURL('note.png')
+            image = staticImageURL('songs/{difficulty}.png'.format(difficulty=difficulty))            
```

Comment:
`staticImageURL` is also here to make it easier and allow you to avoid formatting yourself, so you can do:

```python
image = staticImageURL(difficulty, folder='songs', extension='png')
```